src/utility/TransactionMapping.ts

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
folder hierarchy should be consistent with the [nemtech/nem2-sdk-java](https://github.com/nemtech/nem2-sdk-java).

better would be to introduce the `utility` folder as `src/core/utils` and have `TransactionMapping` added in there

Reply:
sure, will change it